queries/java/context.scm

```diff
@@ -0,0 +1,23 @@
+(if_statement
+  consequence: (_) @context.end
+) @context
+
+(method_declaration
+  body: (_) @context.end
+) @context
+
+(for_statement
+  body: (_) @context.end
+) @context
+
+(enhanced_for_statement
+  body: (_) @context.end
+) @context
+
+(class_declaration
+  body: (_) @context.end
+) @context
+
+([
+  (expression_statement)
+] @context)
```

Comment:
```suggestion
(expression_statement) @context
```

Reply:
Oops didn't notice this. Fixed it myself